theme/theme.config

```diff
@@ -87,11 +87,13 @@
          Import Theme
 *******************************/
 
-@import "theme.less";
+@import (multiple) "theme.less";
```

Comment:
The two changes here would need to be duplicated to every targets theme/theme.config due to changes in less